bytecomp/bytegen.ml

```diff
@@ -345,11 +345,11 @@ let comp_primitive p args =
   | Pfloatcomp Cgt -> Kccall("caml_gt_float", 2)
   | Pfloatcomp Cle -> Kccall("caml_le_float", 2)
   | Pfloatcomp Cge -> Kccall("caml_ge_float", 2)
-  | Pstringlength -> Kccall("caml_ml_string_length", 1)
-  | Pstringrefs -> Kccall("caml_string_get", 2)
-  | Pstringsets -> Kccall("caml_string_set", 3)
-  | Pstringrefu -> Kgetstringchar
-  | Pstringsetu -> Ksetstringchar
+  | Pstringlength | Pbyteslength -> Kccall("caml_ml_string_length", 1)
```

Comment:
I will address the comments you mentioned above, but I don't think it would be complete.
Everytime I add a new primitive I need do a bootstrapping, it takes me lots of time,  my suggestion is @hhugo could take it from here to make it complete for bytecode backend, this is a good step in anyway, right?